example/src/types/MetadataTypes.ts

```diff
@@ -0,0 +1,8 @@
+export type TrackMetadataType = {
```

Comment:
do we need to add `Type` suffix here? isn't it obvious that a type is a type?

Reply:
Removed.